src/app/services/sources/sourceService.ts

```diff
@@ -102,8 +102,24 @@ export class sourceService {
 	}
 
 	async getHtml(link: string): Promise<HTMLHtmlElement> {
+		console.log(link);
 		const stringHtml: string = await cloudscraper(
-			{ method: "GET", url: link },
+			{ method: "GET", url: link,  rejectUnauthorized: false },
+			(error, response, novelHtmlString) => {
+				return novelHtmlString;
+			}
+		);
+
+		const html = document.createElement("html");
+		html.innerHTML = stringHtml;
+		return html;
+	}
+
+
+	async postHtml(link: string): Promise<HTMLHtmlElement> {
+		console.log(link);
+		const stringHtml: string = await cloudscraper(
+			{ method: "POST", url: link, rejectUnauthorized: false},
```

Comment:
this is to prevent cloudscrapper from validating the cert. otherwise boxnovel just dont work